src/components/application_manager/src/commands/hmi/sdl_activate_app_request.cc

```diff
@@ -38,6 +38,43 @@ namespace application_manager {
 
 namespace commands {
 
+namespace {
+struct SDL4AppsOnDevice
```

Comment:
@dev-gh For the people who is not aware of SDL  it will be ambiguous. May be add undersore or something like this: `SDLv4_AppsOnDevice`. Because `4` can be treated as `for` or just as number of apps but not SDL version.